### ISSUE TITLE
Firefox Sync error

### ISSUE THREAD

**Issue by shelterx** [odebroqueville/contextSearch#50]
Any idea what's going on here?
I do run my own sync server but I don't think that's the issue.
`1586636303718   Sync.Engine.Extension-Storage   ERROR   Syncing olivier.debroque               ville@gmail.com: request failed: Error: HTTP 507 Insufficient Storage: Resource                access is forbidden for this user (Maximum bytes per object exceeded " "(31739 >                16384 Bytes.)(resource://services-common/kinto-http-client.js:2771:5) JS Stack`

**Comment by cjscheller** [comment]
Hey @shelterx looks to be an issue related to storing too large of objects into browser sync storage - in this case, with your personal sync server. 

Couple questions: 
- Have you used this extension previously and been able to sync without issues?
- Is this a new bug that has just shown up for you, and if so when was the first time you saw it (and if possible, on what extension version)?
- Any idea at what point in using the extension this error is generated - at install, when saving the options page, etc?

**Comment by shelterx** [comment]
To be honest, I just happend to see it when I ran Firefox from console on Linux. The output showed up there and I reacted when I saw your e-mail address so I looked it up. I thought perhaps the addon was doing something awkward.
On Windows if I turn on sync-log it says:
`1586712943779	Sync.SyncScheduler	DEBUG	Next sync in 3600000 ms. (why=schedule)
1586712944472	Sync.Engine.Extension-Storage	ERROR	Syncing olivier.debroqueville@gmail.com: request failed: TypeError: NetworkError when attempting to fetch resource. No traceback available
1586712944472	Sync.Engine.Extension-Storage	WARN	Syncing failed: TypeError: NetworkError when attempting to fetch resource. No traceback available
1586712944472	Sync.Status	DEBUG	Status for engine extension-storage: error.engine.reason.unknown_fail
1586712944472	Sync.Status	DEBUG	Status.service: success.status_ok => error.sync.failed_partial
1586712944472	Sync.ErrorHandler	DEBUG	extension-storage failed: TypeError: NetworkError when attempting to fetch resource. No traceback available
1586712944473	Sync.Doctor	INFO	Skipping consultation: telemetry reporting is disabled`
